server/core/admin.cc

```diff
@@ -271,6 +272,35 @@ std::string get_filename(const HttpRequest& request)
     return path;
 }
 
+static const char *get_ssl_version(const enum admin_ssl_version_t ssl_version)
+{
+    const char *ver;
+
+    if (ssl_version == ADMIN_SSL_VERSION_NORMAL)
+    {
+        ver = "NORMAL:-VERS-SSL3.0";
+    }
+    else if (ssl_version == ADMIN_SSL_VERSION_TLS10)
+    {
+        ver = "NORMAL:-VERS-SSL3.0";
+    }
+    else if (ssl_version == ADMIN_SSL_VERSION_TLS11)
+    {
+        ver = "NORMAL:-VERS-SSL3.0:-VERS-TLS1.0";
+    }
+    else if (ssl_version == ADMIN_SSL_VERSION_TLS12)
+    {
+        ver = "NORMAL:-VERS-SSL3.0:-VERS-TLS1.0:-VERS-TLS1.1";
+    }
+    else
+    {
+        ver = "NORMAL:-VERS-SSL3.0:-VERS-TLS1.0:-VERS-TLS1.1:-VERS-TLS1.2";
+    }
+
+    MXS_NOTICE("SSL PRIORITY for REST API set to %s", ver);
```

Comment:
Don't log this, it's not useful for the end user. For developers it's probably easier to manually verify by attaching a debugger.